src/main/java/net/glowstone/i18n/LocalizedString.java

```diff
@@ -8,4 +9,6 @@
     String get(Object... args);
 
     void send(CommandSender recipient, Object... args);
+
+    void sendInColor(CommandSender recipient, ChatColor color, Object... args);
```

Comment:
I would prefer if we also supported multiple color codes within the message with the section sign (`§`).

But perhaps it's out of the scope of this PR.

Reply:
For multiple colors, you'd want to add the color-codes to the localizable string anyway, to indicate which part was which color.

Reply:
Yes, this is what I meant. Does that work already?